manual/README.md

```diff
@@ -146,3 +146,50 @@ The following setup assumes a Unix-like environment.
     <br/><br/>
 
 12. Apply to PoA. Please contact the Mezo team for further instructions.
+
+## Starting a node from a CometBFT snapshot
+
+To start a node from a snapshot is no different to any other CometBFT based blockchain.
+
+To start with select a block from a trusted node, you can do so using the following API:
+`http://<NODE_OPERATOR_ADDRESS>:26657/block`, this will give you the very last block
+executed by the node.
```

Comment:
```suggestion
To start with, select a block from a trusted node. You can do so using the following API:
`http://<NODE_ADDRESS>:26657/block`, this will give you the very last block
executed by the node.
```